.gitignore

```diff
@@ -64,3 +64,13 @@ buck-out/
 
 ########### Bundle artifact ###########
 *.jsbundle
+
+########### IntelliJ IDEA ###########
```

Comment:
These things shouldn't be in the project-level `.gitignore`, but in your personal one that's global to your dev machine. (Also see the comment at the top of this file.)

Reply:
Sorry... I have removed unwanted commit and rebased remaining one.